components/03-modules/form/form--compact.hbs

```diff
@@ -0,0 +1,9 @@
+<form class="form-compact {{ class }}">
```

Comment:
The styles will look a bit more complex and specificity will be a bit higher, but then adding a single class in html markup we will be able to change the look of the whole form.